gateway/src/management_switch.rs

```diff
@@ -212,9 +214,14 @@ impl ManagementSwitch {
         let mut port_to_desc = Vec::with_capacity(config.port.len());
         let mut port_to_ignition_target = Vec::with_capacity(config.port.len());
         let mut interface_to_port = HashMap::with_capacity(config.port.len());
+        let retry_config = SpRetryConfig {
```

Comment:
...or, perhaps the whole `SpRetryConfig` could implement `Deserialize`? but, again, the current thing is fine and none of my nitpicks actually matter...

Reply:
I like both these ideas. Thanks for the nitpicks! Will give that a shot shortly.

Reply:
In hindsight, implementing `Deserialize` on `SpRetryConfig` is kinda annoying because it has a `Duration` field, which isn't awesome in TOML. I went with something more like your first suggestion in https://github.com/oxidecomputer/omicron/pull/6575/commits/19facb44ab0d0b3109c38fbd27c6a8deb0a77ae6